decidim/decidim
# Add proposal state label in diff renderer

#### :tophat: What? Why?
Add a label for diff attributes.

#### :pushpin: Related Issues
*Link your PR to an issue*
- Fixes  #12658

#### Testing
1. Login as admin, visit any proposals page 
2. Answer the proposal and change the state 
3. Visit the proposal changeset ( diff render) 
4. See the attribute title is displayed like `Translation missing: ...`
5. Apply the page 
6. Refresh the page

### :camera: Screenshots
![image](https://github.com/decidim/decidim/assets/105683/94413942-3d89-4847-b184-592c4315771a)

![image](https://github.com/user-attachments/assets/5a8e9555-4c37-4d87-ab5f-3d088064c950)


:hearts: Thank you!
